hoot-services/src/main/java/hoot/services/models/osm/Map.java

```diff
@@ -687,7 +687,7 @@ public boolean isVisibleTo(Users user) {
 
         Long ownerId = t.get(maps.userId);
         Long folderId = t.get(folderMapMappings.folderId);
-        Boolean isPublic = t.get(maps.publicCol);
+        Boolean isPublic = t.get(folders.publicCol);
```

Comment:
This was the :ant: causing my user visibility checks to not work.  Maps also have a public column but it is unused and is always null which the conditional below considers to be visible to anyone.